src/Extensions/Logging/Internal/OmexLoggerProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class OmexLoggerProvider : ILoggerProvider, ISupportExternalScope
+	{
+		public OmexLoggerProvider(ILogEventSender logsEventSource, IExternalScopeProvider defaultExternalScopeProvider)
+		{
+			m_logsEventSource = logsEventSource;
+			m_defaultExternalScopeProvider = defaultExternalScopeProvider;
+		}
+
+
+		public ILogger CreateLogger(string categoryName) => new OmexLogger(m_logsEventSource, m_externalScopeProvider ?? m_defaultExternalScopeProvider, categoryName);
+
+
```

Comment:
Do you not need to dispose of anything?

Reply:
No, I don't at this point, it's just a requirement of `ILoggerProvider` interface